ac-ffmpeg/src/format/demuxer.rs

```diff
@@ -458,7 +468,31 @@ impl InputFormat {
 
         Some(res)
     }
+
+    pub fn name(&self) -> String {
+        unsafe {
+            let name = ffw_input_format_name(self.ptr);
+
+            CStr::from_ptr(name)
+                .to_str()
+                .expect("invalid format name")
+                .to_string()
+        }
+    }
```

Comment:
There is no need to clone the string.

```suggestion
    pub fn name(&self) -> &str {
        unsafe {
            CStr::from_ptr(ffw_input_format_name(self.ptr))
                .to_str()
                .expect("invalid format name")
        }
    }
```